client/data/card-readers/hooks.ts

```diff
@@ -3,9 +3,13 @@
  * External dependencies
  */
 import { useSelect } from '@wordpress/data';
+import { CardReaders, CardReaderStats } from 'wcpay/types/card-readers';
```

Comment:
This should be in the internal dependencies block.

Reply:
Done